posthog/temporal/batch_exports/squash_person_overrides.py

```diff
@@ -464,15 +464,17 @@ async def manage_table(
         create_table,
         table_activity_inputs,
         start_to_close_timeout=timedelta(minutes=5),
-        retry_policy=RetryPolicy(maximum_attempts=10, initial_interval=timedelta(seconds=20)),
+        retry_policy=RetryPolicy(maximum_attempts=1),
```

Comment:
Stops accidentally creating this table more than once, haven't seen this fail to warrant a retry, but to keep an eye out.